YANG/ccamp/Client-signal-yang/ietf-trans-client-service.yang

```diff
@@ -79,44 +83,71 @@ module ietf-trans-client-service {
     }
   }
 
+  grouping te-topology-identifier {
+    description
+      "description";
+    leaf access-provider-id {
+      type te-types:te-global-id;
+      description
+        "An identifier to uniquely identify a provider.";
+    }
+
+    leaf access-client-id {
+      type te-types:te-global-id;
+      description
+        "An identifier to uniquely identify a client.";
+    }
+
+    leaf access-topology-id {
+      type te-types:te-topology-id;
+      description
+        "Identifies the topology the service access ports belong to.";
+    }
+  }
+
   grouping  client-svc-instance_config {
     description
       "Configuraiton parameters for client services.";
-
     leaf client-svc-name {
       type string;
       description
         "Name of the p2p transport client service.";
     }
+
     leaf client-svc-descr {
       type string;
       description
         "Description of the transport client service.";
     }
-    uses te-types:te-topology-identifier;
-    container src-access-ports {
-      description
-        "Source access port of a client service.";
-      uses client-svc-access-parameters;
-    }
-    container dst-access-ports {
-      description
-        "Destination access port of a client service.";
-      uses client-svc-access-parameters;
-    }
-    list svc-tunnels {
-      key tunnel-name;
-      description
-        "List of the TE Tunnels supporting the client service.";
-      uses client-svc-tunnel-parameters;
-    }
+
+        uses te-topology-identifier;
```

Comment:
```suggestion
    uses te-types:te-topology-identifier;
```
Why the te-types:te-topology-identifier is not used and a new grouping is defined?

https://github.com/haomianzheng/IETF-ACTN-YANG-Model/blob/f112733bffab2bc83ecf123e5298ed0f8a596ca1/YANG/ccamp/Client-signal-yang/ietf-trans-client-service.yang#L86-L106

Reply:
> I have few minor comments which can be easily addressed before merging the pull request
> 
> In addition, please review the alignment of the code indentation: a consistent indentation makes easier to review the code

This is the right comments, and have been together with other problems. Will show up in the later pull.